prototype-starter/app/globals.js

```diff
@@ -0,0 +1,2 @@
+const govukPrototypeKit = require('govuk-prototype-kit')
+const addGlobal = govukPrototypeKit.views.addFunction
```

Comment:
I don't think we should be putting a globals starter in.

Reply:
Removed